kitsune/search/v2/documents.py

```diff
@@ -221,6 +221,20 @@ def get_field_value(self, field, instance, *args):
             instance = instance.question
         return super().get_field_value(field, instance, *args)
 
+    @classmethod
+    def prepare(cls, instance, **kwargs):
+        obj = super().prepare(instance, **kwargs)
+        # add a prefix to the id so we don't clash with QuestionDocuments
+        obj.meta.id = "a_{}".format(obj.meta.id)
```

Comment:
> and always store the Question ID in ES

We can't have answers use their question id as their ES id, because then they'll conflict with that question in ES and with all the rest of that question's answers.

And if we let ES assign its own ids for both questions and answers, we'd need to completely re-architect how we index questions and answers, so ES didn't get a duplicate every time a change was made to a question or answer.